metricflow/model/semantics/linkable_spec_resolver.py

```diff
@@ -699,7 +721,7 @@ def _get_metric_time_elements(self, measure_reference: Optional[MeasureReference
             measure_agg_time_dimension_reference = measure_semantic_model.checked_agg_time_dimension_for_measure(
                 measure_reference=measure_reference
             )
-            defined_granularity = ValidLinkableSpecResolver._get_time_granularity_for_dimension(
+            defined_granularity = self._get_time_granularity_for_dimension(
```

Comment:
That's still a static method, I don't think we need this.

Reply:
This is totally unrelated and was just a piece of cleanup. It's not necessary but it doesn't impact the code at all. The reason I added this was for developer-friendliness. Basically, when I'm searching for uses of the class, this clutters my search results. So I'm curious if you have a reason to object to this syntax?

Reply:
But also I should have left a comment on the PR explaining / maybe made a separate commit for this!

Reply:
@tlento for the sake of getting the feature released I'm going to merge this, but if you have reasons for me to undo this in a follow up commit LMK and I'm happy to!